src/engine/Settings.cs

```diff
@@ -199,6 +199,11 @@ private Settings()
     /// </summary>
     public SettingValue<int> ThreadCount { get; set; } = new(4);
 
+    /// <summary>
+    ///   Sets the maximum number of entities that can be can exist as well as their overall spawn rate
```

Comment:
> as well as their overall spawn rate

Seems suspicious to me. Do I misunderstand our spawn system or? To me the system does not adjust the spawn rates, it will only skip spawning stuff if the limit is reached, so to me this comment is not correct.

Reply:
Mistake on my part as I didn't realize the comment in Constants was describing more than one thing.  I separated and added a separate comment for MAX_SPAWNS_PER_FRAME as a result.